packages/flutter/lib/cupertino.dart

```diff
@@ -13,6 +13,7 @@ export 'src/cupertino/app.dart';
 export 'src/cupertino/bottom_tab_bar.dart';
 export 'src/cupertino/button.dart';
 export 'src/cupertino/colors.dart';
+export 'src/cupertino/cupertino_localizations.dart';
```

Comment:
Don't add 'cupertino_' again in the file names. The package already says cupertino. Just have the prefix in the class names, since those are the things that get re-exported.